File: week17/week17-4.py
class Solution:
    def isMonotonic(self, nums: List[int]) -> bool:
        n=len(nums)
        big,small=0,0
        for i in range(n-1):
            if nums[i]>nums[i+1]:big=1
            if nums[i]<nums[i+1]:small=1
        if big and small:return False
        return True

File: week17/week17-4.cpp
class Solution {
public:
    bool isMonotonic(vector<int>& nums) {
        int n=nums.size();
        int big=0;
        int small=0;
        for(int i=0;i<n-1;i++){
            if(nums[i]>nums[i+1])big=1;
            if(nums[i]<nums[i+1])small=1;
        }
        if(big&&small)return 0;
        return 1;
    }
};